.github/workflows/update-eventuals-bot.yml

```diff
@@ -0,0 +1,157 @@
+          checkName: 'Build'
+          ref: ${{ steps.pr.outputs.pull-request-head-sha }}
+          repo: ${{ inputs.repo_name }}
+          owner: ${{ inputs.owner }}
+          timeoutSeconds: 3600
+
+      # In case checks of PR failed and the issue hadn't been created before the following action creates issue.
+      # See docs: https://github.com/actions-ecosystem/action-create-issue
+      - name: Create an issue
+        if: steps.wait-for-build.outputs.conclusion == 'failure'
+        uses: actions-ecosystem/action-create-issue@v1
+        with:
+          github_token: ${{ secrets.private_repo_access_as_rebot_token }}
+          title: Updating eventuals failed
+          body: |
+            ## Please check the PR for updating eventuals
+            Pull Request URL - ${{ steps.pr.outputs.pull-request-url }}
+      # The "mergequeue-failed" label gets added by the Mergequeue bot when it has previously seen
+      # the mergequeue-ready label on a PR but the checks failed and Mergequeue couldn't merge the PR.
+      # When that happens it removes the "mergequeue-ready" label and adds "mergequeue-failed".
+      # Even if we later add the "mergequeue-ready" label to the PR again, Mergequeue won't merge a PR
+      # with a new "mergequeue-ready" label if it still has an (old) "mergequeue-failed" label on it.
+      # So we must remove the mergequeue-failed label before re-adding mergequeue-ready.
+      # The following action removes the "mergequeue-failed" label if it exist.
+      # If it doesn't exist the action returns error but doesn't fail the whole job (by default).
+      # See docs: https://github.com/actions-ecosystem/action-remove-labels
+      # - name: Remove label "mergequeue-failed"
+      #   if: steps.pr.outputs.pull-request-operation == 'created' || steps.pr.outputs.pull-request-operation == 'updated'
+      #   uses: actions-ecosystem/action-remove-labels@v1
+      #   with:
+      #     number: ${{ steps.pr.outputs.pull-request-number }}
+      #     labels: mergequeue-failed
+      #     repo: ${{ inputs.owner }}/${{ inputs.repo_name }}
+      #     github_token: ${{ secrets.private_repo_access_as_rebot_token }}
+
+      # We add "mergequeue-ready" label to the pull request created or updated in the step
+      # "Create Pull Request or update existing" in order to merge the changes in automatic mode.
+      # See docs of the action: https://github.com/actions-ecosystem/action-add-labels
+      # - name: Add label "mergequeue-ready"
+      #   if: steps.pr.outputs.pull-request-operation == 'created' || steps.pr.outputs.pull-request-operation == 'updated'
+      #   uses: actions-ecosystem/action-add-labels@v1
+      #   with:
+      #     number: ${{ steps.pr.outputs.pull-request-number }}
+      #     labels: mergequeue-ready
+      #     repo: ${{ inputs.owner }}/${{ inputs.repo_name }}
+      #     github_token: ${{ secrets.private_repo_access_as_rebot_token }}
```

Comment:
If you don't need this we should remove it!

Reply:
We need this for auto-merging. I commented this stuff in order to not make this PR do any auto-merge in eventuals-tutorial before you review it. So, I will uncomment this before I merge it.